### ISSUE TITLE
Look carefully at (Drakh) shields

### ISSUE THREAD

**Issue by Zero1627** [Aatu/FieryVoid#931]
Range 0 attack against Drakh Attack Ship by Furies.
Aft shield is enabled, but attack comes from front (front-stbd to be precise).
Yet damage scored is too low, and I see no legal culprit to account for it. 

Best guesses:
1) range 0 shield was somehow effectively extended?...
2) some strange interaction with Advanced Armor?...

**Comment by Zero1627** [comment]
I have tried to repeat the bug, without success.
At the same time I have confirmet that it seems to happen - database log indicates 3-8 damage bracket for furies (which do 5-10 dmg per shot), clearly indicating a -2 modifier.

I have tested (and failed to score illegally):
 - firing through disabled shield
 - firing through destroyed shield
 - firing through shield that was destroyed while being boosted
 - firing while shield in other aspect was active
All tests conducted both at range 0 and 1.
